docs/tutorials/dicom.ipynb

```diff
@@ -0,0 +1,318 @@
43xwxm/F6KuBPM8vc+YPpBLMiIx7YPw+wwZUlMcZDnqT+uwXHByNuZMjbyndiYWRXLQdW7YRh1\n2TAuyZ5ly1NMKSedzJmOjJ2sFjrZYRhOGv3rMIz+WxeMTWARr//qAl7TFEenwzB4+XUx7Pz8fKr3\nfTEMGu/DMF4yTr3OQCWGvX37tj799NPpRc6d7QYHE8M6PLMcWZZTnvjfFQfqeaxqnrXzmDLz15Vn\n41i55ECyWBBgUAKBCeTslt8IT7Hj4nn5pUK7OUVlQU3DxGaQCCzRHtezc64dTKdHPUfeOXUAbCfc\nrB2i3iVBQvG9wJP601E0H7jXSuo+WKlpvxP6rDsjFBw6L6S1sBPpGkDMWyJUT1ng5DBOIl6iRyuf\n+2MZhDZeg2f5MB19/9HR0Sy7aCAzSIzjw+J9jIHlKo2B2zZ/DfwGr3SmUiYMWgk65pN1tltfmBHg\nkqPxg1w6nqRDQ9mFYZkZROcTwxIDoH2Hmx29aQedMpakQ11V0zpGR+3GMDJlmYHlfhyKfRjW4Q3Z\noc5RTQwzju7CMPph/hnDMohIm0Sbzvr4/NHR0dZeYu+DYWAFGThjGPcnhjGWbxPDaLdbrtBh2OXl\n5Sy7nvqSNiJtyS4M6/yF1L1MkNg/SAxzQOP6MsBcKs/CsUrCJqjYMJthOZ/MYLsIkDpZx4QSAwZk\n2lSAbgAAIABJREFUAjqDg9A4EnSx02NPHcF3BEDdRDkuToW7DQOQx5NTLADK0dHDbty3t7fTNFLS\nBCB3Zs9RjjMpKINB3n33YkP3LwEsoyXolmDrCMw8gY52jJyF4xi/oRnOFdOh+bJl2vQTh84i8T+j\nexszG0baRXa4zhlF2jS97dy4QIu3b9/WZ599tuW0UzweHsWm/gSyDtiW2l4yyt0aN4NXGqYlJ/pj\nKOmI+tguDPOShS57lDpftYxhdtJdLGeZseZ6OxT0cQnDkKvEsHEcpyy2DTQBW9LHWO+6wZfEsJQj\n+ruEYej+Eoa5/Q7DcqG7cQkaOGOUuOX6bbiNYeiRX1Hjxet2ilgnawyjHcaP82v74EXq1JX4lJkr\n88rrtIyLtI/dyqSCC8ffvHlT3/3ud/diWAbH9GsJwzjm724sxif4kPdnENkFPPvKs3Cs7A0zoBTy\ndLrszTvCQAAsxDDHxtQZAwMd/XExkWnXAGZlg1mO/MwQK6XbsddvgMvUPHWkwpsuNoa5CZv7Td9M\ncxwM5ritjN7hvVPcpbn/NCaOILmP3ZUTFM1HG5OMHjJLMwxP7+9jOoC6me5cr9cTGDuatDyyHo/6\nAHiiSkDQjuMwDJMzm2l30wk+eEGvpwAyOrq/v68vvviifvNv/s0tPaEZr05yG51zZNl0XzGECVDQ\nvZv+66LBdLY6GfxYio1+4pDP87tqPj2+ZMCr5lutWF8Tw3KpQBod35dTPYdg2JJTSKGf7n9mnmlv\nKfttbEgMW5LnJQx79erV7HqcE2OYMSjHZP1zAER9vs4YBl/GcZw9KW6H2EFH8sEyxP12qqqeMIy1\naMYw1/Hy5cu6vLyc1efd2jebpw1mD8UwnFc7//7vtVbmGzz44osv6od+6Ie2skGJYfRzH4alHUaW\njGH53WGYPz5u7HO7u8qzcKzSM7ShtABnNsMZHoNY3u86EESY53ad/usIR30ZUXGPQcVPQthBYL7f\nqXcDqNugn6YL/QVkESIWqW82D3u/fPrpp7PXRKDYGF3q76YYecwZJaHPVU/rOLz+q2puLLrsXQI9\nfYd+VTWBiKN2O1SOqswjItJxHKd3TjHFx4s9DYzec8e0tbK5z3bs0ilC+d2XdD4Zqzf+dDYM+jkr\nZic6AeOLL76o169fb9E5o7HT09MtQKAtG/k0wBk4pEPgazMKdB0+ZhDNoOljKNA81xLtw7AukLAh\n6n779S7OBKEvKUMu7o8xzIZvCcNoZwnDPGa+jWHcD32MYeP48DJk+nF3d7eFYTgut7e30/5IuzAM\nelfVDMNwrr4Ohjn48tSpMYzrCMD42Hlxezhn1LtaPUyvOmNUNcewdOitZ56e3IdhHd+41/01pph2\ndq68aSv9TQz76quv6tWrV1t4aby4v7+fZGJfcJa2xdekE7vLcVrCsMTCxO8se7dbGIbhe8Mw/INh\nGH5xGIZfGIbhTz4e//eGYfh7wzD834/f3308PgzD8N8Pw/DLwzD8s2EY/sN9bUCMziONvmwRvx3U\naj6dyDdETOeFujMKMrEzsnDK1cC5VMwcp73tBLpd06MzwjlejwHhdvYKEAR4c3NUO6Ep5KajhdYp\n+O6+pHk3DoOsx9EZbH5TX07X5X15vemVzgKGYmmqEfqxKaOdYWjgp4zSsHXK7cgf2uKAuaSc39/f\n19XV1WxDQPMls0/ZjktGaUvH6IcNOcesFynnXf8P0Zdvqnwo/HJZGt8SuPtb/V7EsKrawjBfn1iy\nhGE8vOM2d5U02qmjbjdloZtWc7FRX8IwZ+aYPrNR/qYwLMds2i1hGEGf8SR5537swjA7MsaqXRjG\nB8eTT4dhXiQPL2mP/uTsSdLQzhr32aHMmZLE5M1ms4VhaS+cvbccmVau8xAMY/zZt45f7pPtsse5\nVA7Zx+quqv70OI4/XlW/u6p+ahiGH6+q/6qq/v44jj9WVX//8X9V1X9SVT/2+PnJqvqLB7QxE/qO\nSHkcj9YOSDIkjaefQEtiWWlgpiMa6r25uan1el3r9XoS2IwQrRBWyFT0FBK34ykvQMbbHLgNP1LM\nKxP8mgg7f4DAMDwtgk0vHYWhzzl3zji99suRsuuiZBQHrXPM9Bn+OqLA8TDoUgdjPj09rZOTk2lX\nZ8ZDVHl9fT2tpfJ6EBwmdninPxmdMxYvtK16mMpdr9cTbWmfHa+9u7NBlXtT3nNfLcZPP66vryfn\n2KBgMLIDhxwZvJZ0MOs0Lxmvo3RPsXOtnSr3n9
```

Comment:
We typically don't save the output cells in a notebook because it bloats the repo and makes for bad diffs. This will display on the site because we execute the notebook.
If you want to display it here, you can save it in an `./images` directory and use a relative path to include.
@MarkDaoust ?

Reply:
@lamberta Updated the PR and removed the output cells.